client.go

```diff
@@ -50,6 +54,9 @@ type Client interface {
 
 	// GetAudience returns the allowed audience(s) for this client.
 	GetAudience() Arguments
+
+	// GetMayAct returns the allowed exchange actor(s) for this client
+	GetMayAct() Arguments
```

Comment:
I think you should move this to a new interface, like `TokenExchangeClient`?

Reply:
Do you mean by extending the Client interface?
What would be the advantage of that?

Reply:
No, not changing the `Client` interface, but creating a new interface. Otherwise any existing implementation of `Client` interface will break because you added an additional method.

Reply:
I believe you miss understood me, I meant a new interface that extends Client.

Reply:
I agree that it is easier to change the interface, but that does bring issues with existing implementations of Client interface. If I upgrade to a new version of fosite with this change, my local Client implementation will not work anymore. Is this acceptable? The fix is trivial though (just implement this extra method).

Reply:
I agree, the fix is trivial, but will have several implications on the remaining implementation. I'll check on those.

Reply:
Modified as suggested.

Reply:
sorry, made a mistake with the naming, will correct.